src/Pages/SignUpPage/SignUpPage.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { TextField, Button, Typography } from '@mui/material';
+import './SignUpPage.css';
+import NavBar from '../../components/NavBar';
+
+const SignUpPage: React.FC = () => {
+    const [name, setName] = useState('');
+    const [username, setUsername] = useState('');
+    const [password, setPassword] = useState('');
+
+    const handleLogin = () => {
+        console.log('Logging in with:', { username, password });
+    };
+
+    return (
```

Comment:
We need to have an email field or a dob field to confirm the user is 18 years or older. Or we could have a terms and conditions acceptance toggle button like in the image below.

![image](https://github.com/mfsl-capstone/mfsl-frontend/assets/74443616/12edc052-cc98-4c49-a23d-7f282bc5e818)

Reply:
Alternatively, some websites that encounter an individual visiting them for the first time, have an age confirmation dialogue box before they can even have access to the website. Like in the image I attached below. We could do something like that as well.

![image](https://github.com/mfsl-capstone/mfsl-frontend/assets/74443616/f459b64b-2b68-4c70-87de-60d20f79f3ea)

Reply:
So I talked to the backend team, and we actually don't need one. A Fantasy Game is legal to play at any age so it is not required to check their age. But if we do have terms and conditions apart from that I could definitely  add that component.

Reply:
Okay perfect, I think we should still have a component for agreeing to Terms and Conditions. Those would be related to use and privacy policies. Just add a component for it right now and we'll populate it with our actual conditions and so on later on during the end of the development phase.